source/1_engine/rex_engine/include/rex_engine/filesystem/vfs.h

```diff
@@ -6,5 +6,5 @@ namespace rex
 }
 
 #ifdef REX_PLATFORM_WINDOWS
-#include "rex_engine/platform/win/filesystem/vfs.h"
+#include "rex_engine/platform/win/filesystem/win_vfs.h"
```

Comment:
Think I already commented on this before but just as a reminder, I think this has to be win/vfs instead of win_vfs

Reply:
I commented in the redesign_vfs PR. I don't think it should be win/vfs